### DISCUSSION TITLE
alist挂载115网盘打开无法显示内容

### DISCUSSION THREAD

**Discussion by lxtdd** [AlistGo/alist#7303] [General]
alist挂载115网盘，首页显示work,但是打开时网盘空白，且提示failed get objs: failed to list objs: invalid character 'a' looking for beginning of value 请问是什么原因？

**Comment by renj567** [comment]
> alist挂载115网盘，首页显示work,但是打开时网盘空白，且提示failed get objs: failed to list objs: invalid character 'a' looking for beginning of value 请问是什么原因？

用新版本
